server/repository/src/db_diesel/changelog_row.rs

```diff
@@ -65,12 +65,11 @@ impl<'a> ChangelogRowRepository<'a> {
         ChangelogRowRepository { connection }
     }
 
-    /// # Arguments:
-    /// * earliest the first cursor to be included in the returned
-    /// * limit the number of entries to be returned
     pub fn changelogs(
         &self,
+        // Earliest the first cursor to be included in the returned
```

Comment:
```suggestion
        // The first cursor to be returned
```

Reply:
changed in https://github.com/openmsupply/open-msupply/commit/45c00a876a925c2c9d597f913244ab80d23bde7e, i guess it's technically not the first returned cursor (we just start the query with that cursor)